state/address_test.go

```diff
@@ -391,6 +382,11 @@ func (s *ControllerAddressesSuite) TestWatchAPIHostPortsForAgents(c *gc.C) {
 	wc.AssertClosed()
 }
 
+func (s *ControllerAddressesSuite) controllerConfig(c *gc.C) controller.Config {
```

Comment:
This doesn't need to accept a `*gc.C`.
Not sure you need a wrapper for a one-liner anyway.

Reply:
I'll remove these, they did do something at one point and I managed to factor that out.